pkgs/development/libraries/aws-c-common/default.nix

```diff
@@ -11,6 +11,12 @@ stdenv.mkDerivation rec {
     sha256 = "0a7hi4crnc3j1j39qcnd44zqdfwzw1xghcf80marx5vdf1qdzy6p";
   };
 
+  # It disables it with MSVC, but CMake doesn't realize that clang for windows,
+  # even with gcc-style flags, has the same semantic restrictions.
```

Comment:
Is aws manually adding fpic? Perhaps they have a flag to disable it.

Reply:
They are manually adding it for non-`MSVC`. CMake should have a better notion of `MSVC` / destinguish between MSVC the abi (which we are) and MSVC which we aren't since the wrappers just know gcc-style flags.